lms/djangoapps/ccx/tests/test_utils.py

```diff
@@ -185,6 +187,7 @@ def test_auto_enroll_passed_correctly(self):
 class TestEnrollEmail(ModuleStoreTestCase):
     """tests for the enroll_email function from ccx.utils
     """
+    MODULESTORE = TEST_DATA_SPLIT_MODULESTORE
```

Comment:
I don't know.  We put it here to ensure that the course created will have a split-mongo id, but I suppose that's not a hard requirement, since these tests do not try to create a CCXLocator at any point.  

On the other hand, being consistent in the modulestore used for tests might be helpful in documenting that ccx is not compatible with the legacy mongo implementation.  I'm certainly open to changing that opinion, though @cpennington